cmd/fleetctl/fleetctl/api.go

```diff
@@ -289,11 +311,15 @@ func apiCommand() *cli.Command {
 				method = flMethod
 			}
 
+			if body != nil && method == "GET" {
+				return fmt.Errorf("GET requests may not include a body")
+			}
+
 			if !strings.HasPrefix(uriString, "/") {
 				uriString = fmt.Sprintf("/%s", uriString)
 			}
 
-			if !strings.HasPrefix(uriString, "/api/v1/fleet") {
+			if !strings.HasPrefix(uriString, "/api/v1/fleet") && !strings.HasPrefix(uriString, "/api/latest/fleet") {
```

Comment:
RFC: are there any other prefixes we should exclude here? Or maybe it should just be `strings.HasPrefix(urlString, "/api/")`.